app/views/user_mailer/admin_hidden_work_notification.html.erb

```diff
@@ -1,12 +1,11 @@
 <% content_for :message do %>
-    <p>Dear <%= @user.default_pseud.byline %>,</p>
+    <p><%= t('user_mailer.admin_hidden_work_notification.hello', user: @user.default_pseud.byline) %></p>
 
-    <p>Your work <i><%= style_link(@work.title.html_safe, work_url(@work)) %></i> has been reported as being in violation of the Archive of Our Own's <%= style_link("Terms of Service", tos_url) %>.</p>
+    <p><%= t('user_mailer.admin_hidden_work_notification.html.part1', title: style_link(@work.title.html_safe, work_url(@work)), tos: style_link('user_mailer.admin_hidden_work_notification.tos', tos_url).html_safe).html_safe%> 
```

Comment:
The double .html_safe is needed as one is marking style_link('user_mailer.admin_hidden_work_notification.tos', tos_url) as safe while the other is marking the whole string.